cupy/cuda/compiler.py

```diff
@@ -1,36 +1,34 @@
 import hashlib
 import os
 import re
-import subprocess
-import sys
 import tempfile
 
 import filelock
 import six
 
 from cupy.cuda import device
 from cupy.cuda import function
+from pynvrtc.compiler import NVRTCInterface
+from pynvrtc.compiler import Program
 
+_nvrtc_version = None
 
-_nvcc_version = None
 
+def _get_nvrtc_version():
+    global _nvrtc_version
```

Comment:
It uses global variable. When a user use this method from multiple threads, what happens? If it is safe, it's OK.